greenboot.spec

```diff
@@ -90,6 +90,9 @@ install -DpZm 0644 etc/greenboot/greenboot.conf %{buildroot}%{_sysconfdir}/%{nam
 %systemd_post greenboot-grub2-set-success.service
 %systemd_post greenboot-rpm-ostree-grub2-check-fallback.service
 %systemd_post redboot-auto-reboot.service
+if [ -d /usr/lib/bootupd/grub2-static/configs.d ]; then
```

Comment:
Nope! This is not going to work.

Reply:
@nullr0ute do you have any suggestions for conditionalizing where the grub config lands as Sayan is trying to do?

Reply:
The general rule of thumb is that you shouldn't do it. I think we need to step back and look at the actual problem.

Reply:
So I applied a hard link instead of mv also added in %postun to remove the ln post installation.

Reply:
What about creating a symlink to the file instead of moving it? 
This way we can also undo the symlink creation in the %postun section

Reply:
Is there a chance that /etc would be mounted on a different flesystem? 
Is symlink better for what we're trying to do?

Reply:
Thats a good point, hardlink will have issue between different file systems, softlink didn't work so I plan on using `cp` as it may not have an integrity issue as pointed out by @nullr0ute